ts/features/idpay/initiative/configuration/screens/IbanConfigurationLandingScreen.tsx

```diff
@@ -46,8 +40,10 @@ const IbanConfigurationLanding = () => {
         <View
           style={[IOStyles.horizontalContentPadding, styles.adjustCenterHeight]}
         >
-          <View style={styles.greyCircle}></View>
-          <VSpacer size={24} />
+          <IbanPlaceholderSVG height={150} width={"100%"} />
+          <VSpacer size={32} />
+          <VSpacer size={16} />
+          <View style={{ alignSelf: "center" }} />
```

Comment:
```suggestion
          <IbanPlaceholderSVG height={240} width={240} />
          <VSpacer size={24} />
```

Reply:
if I'm not wrong on smaller devices we scale the size to better fit the content